apps/arena/lib/arena/game_updater.ex

```diff
@@ -51,12 +51,18 @@ defmodule Arena.GameUpdater do
   # END API
   ##########################
 
-  def init(%{clients: clients, bot_clients: bot_clients}) do
+  def init(%{clients: clients, bot_clients: bot_clients, game_params: game_params}) do
     game_id = self() |> :erlang.term_to_binary() |> Base58.encode()
     game_config = Configuration.get_game_config()
+    game_config = Map.put(game_config, :game, Map.merge(game_config.game, game_params))
+
     game_state = new_game(game_id, clients ++ bot_clients, game_config)
     match_id = Ecto.UUID.generate()
 
+    unless game_config.game.bots_enabled do
+      Process.send_after(self(), :toggle_bots, 300)
```

Comment:
Why this 300? is this to give time to the bots to init? if that's the case i think this is too error prone, maybe we could think in another approach, maybe sending something in the bot init message when the match start to they wake up sleep.

Reply:
Yeah, the 300 ms seems random. I moved it to the game_start event.
Moving it to the init bot ws message would be a longer solution since we don't read the config there. I believe it's fine and faster just do it in the game start, since we have the config there.